README.md

```diff
@@ -124,18 +124,18 @@ report.
 Below is a table of CML functions for writing markdown reports and delivering
 those reports to your CI system.
 
-| Function                | Description                                                      | Example Inputs                                              |
-| ----------------------- | ---------------------------------------------------------------- | ----------------------------------------------------------- |
-| `cml runner`            | Launch a runner locally or hosted by a cloud provider            | See [Arguments](https://github.com/iterative/cml#arguments) |
-| `cml publish`           | Publicly host an image for displaying in a CML report            | `<path to image> --title <image title> --md`                |
-| `cml send-comment`      | Return CML report as a comment in your GitLab/GitHub workflow    | `<path to report> --head-sha <sha>`                         |
-| `cml send-github-check` | Return CML report as a check in GitHub                           | `<path to report> --head-sha <sha>`                         |
-| `cml pr`                | Commit the given files to a new branch and create a pull request | `<path>...`                                                 |
-| `cml tensorboard-dev`   | Return a link to a Tensorboard.dev page                          | `--logdir <path to logs> --title <experiment title> --md`   |
+| Function                  | Description                                                      | Example Inputs                                              |
+| ------------------------- | ---------------------------------------------------------------- | ----------------------------------------------------------- |
+| `cml runner launch`       | Launch a runner locally or hosted by a cloud provider            | See [Arguments](https://github.com/iterative/cml#arguments) |
+| `cml publish`             | Publicly host an image for displaying in a CML report            | `<path to image> --title <image title> --md`                |
```

Comment:
```suggestion
```

Reply:
do we want to take it out completely? I know this has been pretty nicely built into `comment`. Do we want to mention the hidden alias of `asset`? It is the most used command.

Reply:
We should probably not document deprecated subcommands.

Reply:
I'd prefer not to mention any hidden commands